torch/fx/experimental/symbolic_shapes.py

```diff
@@ -4900,6 +4959,15 @@ def track_symfloat(source: Source, val: Union[float, SymFloat]) -> None:
                         curr_t.storage_offset(),
                     )
 
+                    if (
+                        isinstance(curr_t, torch._subclasses.fake_tensor.FakeTensor)
+                        and curr_t.try_get_nested_int_id() is not None
+                    ):
+                        track_symint(
+                            torch._dynamo.source.NestedIntSource(src),
+                            torch._nested_int_from_offsets(curr_t),
+                        )
```

Comment:
This is pointless because it will always create a fresh SymInt. In the case where we do an equality, (1) we don't complain that we've encountered an untracked symbol, and (2) no guard is created. This may be problematic in the case where we have two tensors that have the same id in the registry due to .to() - a preexisting issue.